src/components/DataControllerMixin/DataControllerMixin.js

```diff
@@ -74,56 +74,39 @@ var ModalMixin = {
     return data;
   },
 
-  // TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/124
-  //Fetches and validates data from a call to the backend's
-  //"multistat" API endpoint
-  getStatsPromise: function (props, timeidx) {
-    return axios({
-      baseURL: urljoin(CE_BACKEND_URL, 'multistats'),
-      params: {
-        ensemble_name: props.ensemble_name,
-        model: props.model_id,
-        variable: props.variable_id,
-        emission: props.experiment,
-        area: props.area || null,
-        time: timeidx,
-      }
-    }).then(validateStatsData);
+  //Returns the metadata object that corresponds to a unique_id
+  getMetadata: function (id, meta = this.props.meta) {
+    return _.find(meta, function(m) {return m.unique_id === id;} );
   },
-
-    //Returns the metadata object that corresponds to a unique_id
-    getMetadata: function (id, meta = this.props.meta) {
-      return _.find(meta, function(m) {return m.unique_id === id;} );
-    },
     
-    /*
-     * Filters data from any call to the API that returns an object with 
-     * individual values keyed to unique_ids. It returns a new object 
-     * that contains only results from datasets whose metadata matches the 
-     * attributes passed to the filter argument. 
-     * 
-     * This is a temporary stopgap until this issue is solved:
-     * https://github.com/pacificclimate/climate-explorer-backend/issues/61
-     * FIXME: remove this function and calls to it when issue 61 is solved.
-     */
-    filterAPIResults: function(data, filter, metadata = this.props.meta) {
-      var filtered = {};
+  /*
+   * Filters data from any call to the API that returns an object with
+   * individual values keyed to unique_ids. It returns a new object
+   * that contains only results from datasets whose metadata matches the
+   * attributes passed to the filter argument.
+   * 
+   * This is a temporary stopgap until this issue is solved:
+   * https://github.com/pacificclimate/climate-explorer-backend/issues/61
+   * FIXME: remove this function and calls to it when issue 61 is solved.
```

Comment:
https://github.com/pacificclimate/climate-explorer-backend/issues/61 is marked as closed by https://github.com/pacificclimate/climate-explorer-backend/pull/67, so this function should be removed as per FIXME.

Reply:
Good catch! One step closer to getting rid of DataControllerMixin entirely. :)